_docs/operations/users.md

```diff
@@ -72,3 +72,18 @@ To delete a user, run:
 ```bash
 $ storageos user rm jim
 ```
+## Altering the StorageOS API account
```

Comment:
This only applies to native driver install. CSI doesn't have this caveats. I think we should specify.